pkg/workloads/console/controller.go

```diff
@@ -74,115 +74,113 @@ func Add(ctx context.Context, logger kitlog.Logger, mgr manager.Manager, opts ..
 	return ctrl, err
 }
 
-type ConsoleReconciler struct {
+type Controller struct {
 	ctx      context.Context
 	logger   kitlog.Logger
 	recorder record.EventRecorder
 	client   client.Client
 }
 
-func (r *ConsoleReconciler) Reconcile(request reconcile.Request) (res reconcile.Result, err error) {
-	logger := kitlog.With(r.logger, "request", request)
+func (c *Controller) Reconcile(request k8rec.Request) (k8rec.Result, error) {
+	logger := kitlog.With(c.logger, "request", request)
 	logger.Log("event", EventStart)
 
-	defer func() {
```

Comment:
How come no defer? I was in favour of the defer to ensure we always log the error, and could omit the logging from any place that generated one.

Reply:
We now log the error in the outer `Controller` - the `Reconciler` just returns the error. Same result (I think) - slightly cleaner?

Reply:
Ah gotcha, my mistake. 👍